### DISCUSSION TITLE
^K in the search field

### DISCUSSION THREAD

**Discussion by dietmar-schoder** [scalar/scalar#3554] [General]
Is this "^K" in the search field deliberate?

![Screenshot 2024-10-19 120134](https://github.com/user-attachments/assets/5dd17afc-427c-447c-9c7c-5726c89729c3)

**Comment by marclave** [comment]
it's meant to be the META key (CMD on macos) + K to open the search modal like below

https://github.com/user-attachments/assets/99aad314-b613-4b8f-891f-c01d5b25f51a

does this work for you (CNTRL+K) @dietmar-schoder

**Reply by dietmar-schoder** [reply]
Oh, it means "Ctrl-K" on a Windows machine! LOL.

Sorry for asking a stupid question.

**Comment by dietmar-schoder** [comment]
Thank you for the prompt explanation.
